src/parser/Clauses.h

```diff
@@ -259,6 +259,29 @@ class OverEdges final {
     std::vector<std::unique_ptr<OverEdge>> edges_;
 };
 
+class FetchLabels final {
+public:
+    void addLabel(std::string *label) { labels_.emplace_back(label); }
+
+    std::vector<std::string *> labels() {
+        std::vector<std::string *> result;
+        std::transform(labels_.cbegin(), labels_.cend(),
+                       std::insert_iterator<std::vector<std::string *>>(result, result.begin()),
+                       [](auto &label) { return label.get(); });
+        return result;
+    }
+
+    std::string toString() const;
+
+    std::string* release() {
```

Comment:
Weird about this method. Could you add some comments about it.

Reply:
remove it.